README.md

```diff
@@ -64,7 +64,7 @@ just remember that you can't purchase success!
 
 # Curriculum
 
-The curriculum is separated into two parts:
+Check out our curricular guidelines [here](CURRICULAR_GUIDELINES.md).
```

Comment:
We definitely want our guidelines to be accessible when users are interested in digging in to them. And before that we want users to be aware that they exist. But this language suggests that the first thing readers should do when they start to review what makes up the curriculum is to check out the curricular guidelines.

Reply:
After making my pull request, I noticed that the changes made the line referring to the separation of the curriculum into two parts seem redundant, maybe it would be better to remove it to avoid confusion and make the guidelines more accessible for readers. I will make that change in the next version of the draft.